lib/ansible/modules/storage/netapp/na_ontap_svm.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+
+module: na_ontap_svm
```

Comment:
It seems like "SVM" is the preferred nomenclature for this- why is `vserver` used everywhere in the module args and docs?